packages/utils/src/route.js

```diff
@@ -41,29 +41,26 @@ export const flatRoutes = function flatRoutes (router, fileName = '', routes = [
   return routes
 }
 
-function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash) {
-  let start = -1
+function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash, namePrefix) {
```

Comment:
I don't know if the fix is sound in general so you might hold off addressing my comments but... wouldn't it be better to call the new argument `parentRouteName`? A bit easier to understand?